certbot/cli.py

```diff
@@ -836,8 +839,9 @@ def add_plugin_args(self, plugins):
 
         """
         for name, plugin_ep in six.iteritems(plugins):
-            parser_or_group = self.add_group(name,
-                                             description=plugin_ep.long_description)
+            if name != 'certbot-route53:auth':  # Remove duplicate route53 options from help.
+                parser_or_group = self.add_group(name,
+                                                 description=plugin_ep.long_description)
             plugin_ep.plugin_cls.inject_parser_options(parser_or_group, name)
```

Comment:
This should also be in the `if`

Reply:
`parser_or_group` is set to the previous loop's iteration if the first `if` isn't hit.

Reply:
Yes it is, I believe it is set to apache. If groups are not used for anything other than help then does this matter as it is always hidden? I don;t know if it is possible to add it to no group.